src/plugins/presentation_util/public/components/controls/control_group/control_group.scss

```diff
@@ -0,0 +1,186 @@
+$smallControl: $euiSize * 14;
+$mediumControl: $euiSize * 25;
+$largeControl: $euiSize * 50;
+$controlMinWidth: $euiSize * 14;
```

Comment:
Does it make sense to just set `$controlMinWidth: $smallControl` since they are the same value?